### PR TITLE
Add Appveyor CI packaging

### PR DESCRIPTION
**Pull request by tolbrino**
- Disable OTP 21 builds on AppVeyor
- Remove rocksdb deps from aecore app deps

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -76,3 +81,9 @@ deploy: off
 on_finish:
 # Set blockRdp to true to allow RDP, uses Git tag build-agent-v6.1.0%2B1300
   - ps: $blockRdp = $false; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/ab48d6e110f2feb585b6b908a72d6e7dbfb83cb3/scripts/enable-rdp.ps1'))
+
+artifacts:
+  - name: Windows Release
+    path: aeternity-windows-w64-*.exe
```
Comment (dincho) [inline appveyor.yml]: Could you please change the naming to match our current scheme?
It's `aeternity-$version-$osname-$arch-*` Where arch is `x86_64` for 64bit architectures.

Comment (dincho) [reply appveyor.yml]: Of course leave it as is if it's too much hassle

Comment (tolbrino) [reply appveyor.yml]: Changed it.

File: apps/aecore/src/aecore.app.src.script [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+                   case IsWindows of
+                       false ->
+                           Apps;
+                       true ->
+                           Apps -- [rocksdb, mnesia_rocksdb]
+                   end
+                end,
+
+%% Remove rocksdb related dependencies from app runtime dependencies
+[{application, aecore, AppConfig0}] = CONFIG,
```
Comment (lucafavatella) [inline apps/aecore/src/aecore.app.src.script]: List - ok. With one element being term in aecore.app.src - ok.

**Comment by tolbrino** [comment]
@dincho I fixed the package name, see https://ci.appveyor.com/project/aeternity/aeternity/builds/21995367/job/a0fh1c7ns3emvjo0/artifacts

File: ci/appveyor/package.cfg [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+packages = {pkg_prefix}libsodium
+
+filename_stub = aeternity-windows
+
+display_name = aeternity for Windows
+
+description = The full aeternity node for Windows.
+
+version = 1.3.0-alpha.1
```
[the remark this answers is not in the document]
Comment (tolbrino) [reply ci/appveyor/package.cfg]: Yes.

File: .gitignore [hunk 4]
```diff
@@ -57,3 +57,5 @@ data/aecore/keys
 _checkouts/
 tmp/
 rebar3.crashdump
+aeternity*windows*.zip
```
[the remark this answers is not in the document]
Comment (tolbrino) [reply .gitignore]: That could be an improvement.

**Pull request merged**